SPECS/xcp-ng-xapi-plugins.spec

```
@@ -32,6 +32,10 @@ install SOURCES/etc/xapi.d/plugins/xcpngutils/*.py %{buildroot}/etc/xapi.d/plugi
 %dir /var/lib/xcp-ng-xapi-plugins
 
 %changelog
+* Fri May 20 2022 Benjamin Reis <benjamin.reis@vates.fr> - 1.7.2-1
+- Do not escape disabled repos in yum command
+- Fix typo in variable name
```

A changelog that attempts to talk the user language is better:
```suggestion
- Fix repository selection during updates with yum
- Fix exception handling when a command fails
```